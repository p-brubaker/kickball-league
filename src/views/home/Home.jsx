import './Home.css'

export default function Home() {
    return (
        <div className="home">
            <h1>Kickball manager</h1>
        </div>
    )
}
