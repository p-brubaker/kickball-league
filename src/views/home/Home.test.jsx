import { render } from '@testing-library/react'
import { MemoryRouter } from 'react-router-dom'
import Home from './Home'

it('should render the home component', async () => {
    const { container } = render(
        <MemoryRouter>
            <Home />
        </MemoryRouter>
    )

    expect(container).toMatchSnapshot()
})
