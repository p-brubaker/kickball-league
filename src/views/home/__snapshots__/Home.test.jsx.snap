// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the home component 1`] = `
<div>
  <div
    class="home"
  >
    <h1>
      Kickball manager
    </h1>
  </div>
</div>
`;
